tools/rapids-conda-retry

```diff
@@ -23,9 +19,18 @@
 #
 # RAPIDS_CONDA_RETRY_SLEEP     - set to a positive integer to set the duration, in
 #                                seconds, to wait between retries.
-#                                Default is a 10 second sleep
+#                                Default is a 10 second sleep.
+#
+# RAPIDS_CONDA_RETRY_TIMEOUT   - Timeout for each individual retry.
+#                                Positive integers are interpreted as seconds,
+#                                but unit strings like '2h' for "two hours" will also work.
+#                                Default varies based on the command being run.
 #
+
+# This must be set in order for the script to recognize failing exit codes when
+# output is piped to tee
```

Comment:
I think it was a mistake that this got mixed in with the multi-line comment about arguments... seems like it was supposed to specific to the `set -o pipefail` line.